packages/schema/package.json

```diff
@@ -56,32 +83,38 @@
     "build:tsc": {
       "command": "tsc --pretty",
       "files": [
-        "src/",
+        "package.json",
+        "src/**/*.ts",
         "tsconfig.json",
+        "__tests__/**/*.ts",
         "../../core/tsconfig/base.json"
       ],
       "output": [
-        "dist/",
+        "dist/**",
         "tsconfig.tsbuildinfo"
       ]
     },
-    "build:update-json-schema": {
-      "command": "node dist/scripts/update-json-schema.js",
-      "dependencies": [
-        "build:tsc"
+    "build:schema": {
+      "command": "tsx src/scripts/generate-json-schema.ts",
```

Comment:
In general I'd like us to just do tsc -> node. Having a bunch of ways to do tsc compilation seems unnecessary. So, this should depend on buid:tsc, and should just invoke node on the ts files.